### ISSUE TITLE
re: external-ip

### ISSUE THREAD

**Issue by Howardpo** [matryer/bitbar-plugins#1053]
FAO: (@matryer)This no longer seems to be working on my system. I replaced the following and it now seems to work:

Old, non-working line: 
EXTERNAL_IP=$(dig +short myip.opendns.com @resolver1.opendns.com)

New, working line:
EXTERNAL_IP=$(curl -s monip.org | perl -ne '/IP : ([0-9.]+)/ && print $1')

**Comment by brainwahve** [comment]
Not sure about how valid my issue is in that the "dig" works when I'm at my home but wasn't working when I was at a hotel. The "curl" version seems to work in both places.

**Comment by Howardpo** [comment]
The issue with the "dig" style line was when I was at a hotel, that may have been the problem. The "curl" style seems to work in both situations.

**Comment by tresni** [comment]
Yeah, these are getting IP from 2 different places.  The `dig` command gets it based on direct DNS request from you to `resolver1.opendns.com`, the `curl` gets it from an HTTP connection.  Both have merit, but either one can be wrong (e.g. if your HTTP or DNS connection is transparently proxied by your provider, business, etc.)

**Comment by sprak3000** [comment]
Closing due to age and lack of plugin author response.
